01_login.php

```diff
@@ -54,134 +55,132 @@
 
 		<!-- Navbar (sit on top) -->
 		<div class="w3-top">
-			<div class="w3-bar w3-white w3-wide w3-padding w3-card">
+			<div class="w3-bar w3-grey w3-padding w3-card">
 				<img src="images/placeholder.jpg" width="40" height="40">
-				<a class="w3-bar-item"><b>logo</b> </a>
+				<a class="w3-bar-item"><b>Soundwave</b> </a>
+			</div>
+		</div>
 
-			</div></div>
 
+		<!-- background colour -->
+		<div class="w3-black">
+			<div class="w3-container w3-content w3-padding-64" style="max-width:100%">
 
-		<br>
-		<br>
-		<br>
-		<br>
-		<br>
 
+				<!-- Container (placeholder) -->
+				<div class="w3-content w3-container w3-padding-64">
 
-		<!-- logo image placing-->
-		<div class="w3-content w3-container w3-padding-64">
+					<div class="w3-row">
+						<div class="w3-col m6 w3-center w3-padding-large">
+							<img src="images/placeholder.jpg" class="w3-round w3-image w3-opacity" alt="placeholder" width="500" height="333">
+						</div>
 
-			<div class="w3-row">
-				<div class="w3-col m6 w3-center w3-padding-large">
 
-					<img src="images/placeholder.jpg" class="w3-round w3-image w3-opacity w3-hover-opacity-off" alt="Photo of Me" width="500" height="333">
-				</div>
+						<br>
 
+						<!-- brand name: soundwave -->
+						<div class="w3-col m6 w3-padding-large">
+							<h1>SOUNDWAVE</h1>
+							<form method = "post" id= "01_login">
 
-				<!-- brand + opening message -->
-				<div class="w3-col m6 w3-hide-small w3-padding-large">
```

Comment:
removed feature of hide-small so small screens are able to use this website on any device